lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -471,7 +471,9 @@ def check_scope
               send('decrement_positions_on_lower_items') if lower_item
               cached_changes.each { |attribute, values| self[attribute] = values[1] }
 
-              send("add_to_list_#{add_new_at}")
+              if add_new_at.present?
+                send("add_to_list_#{add_new_at}")
```

Comment:
Can you make this a one-liner?

`send("add_to_list_#{add_new_at}") if add_new_at.present?`